index.js

```diff
@@ -222,6 +227,13 @@ function commit(selected: Element, details: Element) {
 function keydown(event: KeyboardEvent) {
   const details = event.currentTarget
   if (!(details instanceof Element)) return
+  const menu = details.querySelector('details-menu')
+  if (!menu) return
```

Comment:
In #42, the details and menu elements are bound arguments to `keydown` so we won't have to query for them here.